mesop/examples/__init__.py

```diff
@@ -19,5 +19,6 @@
 from mesop.examples import sxs as sxs
 from mesop.examples import testing as testing
 from mesop.examples import text_io as text_io
```

Comment:
no action needed: but food for thought, I think `text_to_image` is a good name, but I'm now wondering if `text_io` was a good name 😅 . It may be worth making `text_to_text` an alias for `text_io` later.

Reply:
Yes, I agree. Yeah I think you mentioned text_to_text last time we talked.